### ISSUE TITLE
Keyboard numberpad keys not working except 5

### ISSUE THREAD

**Issue by kazosman** [laurent22/joplin#9240]
The numberpad to the right of the main keyboard doesn't work except for the number 5 key.

## Environment

Joplin version: 2.9.17
Platform: Linux
OS specifics: Ubuntu 22.04 

## Steps to reproduce

1. Launch Joplin
2. Create a note
3. Try using the number keys

## Describe what you expected to happen
I am hoping (and praying) the numberpad keys can be made to work as I use the more than the ones above qwertyuiop.

**Comment by personalizedrefrigerator** [comment]
The number pad keys work for me in Joplin on Ubuntu 23.10 (no plugins).

More information would be helpful in reproducing this:
- Do the number pad keys work in other applications? (If you have Google Chrome/Chromium/Brave or another Chromium-based browser installed, it would be particularly helpful to know if they work in that.)
- If you use plugins, do the number pad keys work after disabling all plugins and restarting Joplin?
- Is this the case for both the rich text and markdown editors?

**Comment by kazosman** [comment]
I have chromium installed and the numberpad is working:
![image](https://github.com/laurent22/joplin/assets/38760282/ac998009-58b9-43c1-941b-ae2ddf85c004)
I disabled the installed plugins (Advanced Tagging Rules and Table Formatter Plugin) and the numberpad started working. I enabled each one on their own and the numberpad works and now with both enabled the numberpad also works. I think mine may be possessed.
I am closing this bug.
Thank you
